whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -1443,7 +1443,13 @@
       "$n": {"about": "_:instance", "addLink": "hasNote", "resourceType": "Note", "property": "label",  "ignoreOnRevert": true},
       "$o": {"NOTE:subfield-count (Libris)": 0, "NOTE": "Cannot ignore this field since we import Mimer data with $o (We have lost $o data between 1.14.0 and 1.15.0). See LXL-1517"},
       "$s": {"about": "_:workTitle", "property": "mainTitle", "TODO:targetMatch": "240"},
-      "$t": {"about": "_:title", "property": "mainTitle", "TODO:targetMatch": "222, 245"},
+      "$t": {
+        "about": "_:title",
+        "property": "mainTitle",
+        "allowedRevertTypes": ["KeyTitle", "Title"],
```

Comment:
For the line of thought in my lengthier comment, this would have to be moved to the declaration of `"_:title"` (at line 1415), where the link and type are specified.

Reply:
`allowedRevertTypes` might logically be better as `allowedSubTypesOnRevert` with just `KeyTitle` here, to omit the redeclaration of `Title` (which is also its base type). But that might be nitpicking over something we may not see used elsewhere  (or theoretically overly restricting if other cases do crop up -- although I suspect that logical restriction ought to apply in such cases too).